src/shell.ts

```diff
@@ -27,27 +28,50 @@ export class Shell {
         this.commands = loadCommands(this)
         this.currentLine = ''
         this.t.scrollToBottom()
+        document.addEventListener('keydown', ev => this.updateCapsLock(ev))
```

Comment:
Should it be keydown or keyup? from trying it out it seems the indicator is turn on the right moment, but turns off only after I click caps lock again and press a key. My guess is that the modifier is still on in keydown and turns off only on keyup.

Reply:
It works properly for me - as long as caps lock is on the indicator is shown. No need to press a key. Maybe iPads handle caps on/off differently?

Reply:
I've tested it on the Mac so it's probably a safari/chrome thing. I'm marking as resolve as on the iPad I map the caps lock to escape as the keyboard has no escape.
If it works for you it's good enough.